### DISCUSSION TITLE
FS22_FlieglTrailerPack by Ifkonator

### DISCUSSION THREAD

**Discussion by Wingman93** [ddewar/FS22_UniversalAutoloadModhubAddon#98] [Q&A]
I need Help,
I Downloaded me the [FS22_FlieglTrailerPack by Ifkonator](https://www.lsfarming-mods.com/filebase/file/194-fliegl-anh%C3%A4nger-pack/) and make these Text edits for UAL.
I put this in the UniversalAutoloadModhubAddon and start the Game with the [FS22_FlieglTrailerPack](https://www.lsfarming-mods.com/filebase/file/194-fliegl-anh%C3%A4nger-pack/), [FS22_UniversalAutoload](https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=237080&title=fs2022) & [FS22_UniversalAutoloadModhubAddon](https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=251255&title=fs2022) ([with these extra Edits](https://workupload.com/file/DQTTGSvCr8f)) and nothing show me Ingame that I can load anything.

![Unbenannt](https://github.com/ddewar/FS22_UniversalAutoloadModhubAddon/assets/135762188/8f91b9eb-ec6a-4e43-8d1c-e6aecb7f67c8)

**Comment by ddewar** [comment]
I'm not sure which edits you made, but have you tried adding them to your modsettings/universalautoload.yml file? That is where we put the personal overrides.

**Reply by Wingman93** [reply]
I write this Text edits in the UniversalAutoloadModhubAddon by supported Vehicles

**Reply by Wingman93** [reply]
The Text edits i make i add to the modsettings/universalautoload.xml file. Now it only shows me loading logs not logs on 1 Option and Bales & Pallets on other Option. Iam Comfused now 😅

**Reply by KingArthurLV** [reply]
So acording to screenshoted lines, there is only 2 supports per trailer size. One is for logs which is option 4 and second is everything else which is option 8. If there are other trailer configurations, in this code snippet there are not support provided for it.

**Reply by Wingman93** [reply]
@KingArthurLV i only will have the Option 4 for Loading Bales and Pallets but also can use option 4,5,6,7 and only option 8 for Logs.
but Problem is all options show me now Loading Logs only. I know the height sizes are not correct at the moment but it only show me 1 option to load not 2 like the Orkel Trailers in Modhub
